apps/astarte_appengine_api/lib/astarte_appengine_api/device/device_status.ex

```diff
@@ -34,6 +34,7 @@ defmodule Astarte.AppEngine.API.Device.DeviceStatus do
     field :first_credentials_request, :utc_datetime
     field :last_credentials_request_ip
     field :last_seen_ip
+    field :metadata, :map
```

Comment:
map type is quite permissive, while we require maps here to be %{string.t() => string.t()}.
@rbino should we use an ecto changeset just for it?